content/master/concepts/composition.md

```diff
@@ -1168,6 +1168,39 @@ not considered to be 'empty', and thus will pass the readiness check.
 
 `None`. Considers the composed resource to be ready as soon as it exists.
 
+### Composition Validation
+
+Crossplane uses a `Validating Webhook` to inform users of any potential
+misconfigurations in a `Composition` as early as possible. The default behavior
+of the webhook is to perform `logical checks` only, enforcing requirements that
+aren't explicitly defined in the schema but are assumed to hold at runtime.
+
+#### Experimental Validation With Schemas
+
+Crossplane provides experimental schema-aware validation that can be enabled
+through the `--enable-composition-webhook-schema-validation` feature flag. This
+will enable Composition validation against available schemas in the cluster,
+ensuring, for example, that `fieldPaths` are valid and source and destination
+types match taking into account provided transforms too.
+
+The `crossplane.io/composition-validation-mode` annotation on the Composition
+allows setting one of two modes for schema validation:
+
+-  `loose` (default): Same as `strict` mode, except that in case of missing
```

Comment:
```suggestion
-  `loose` (default): Same as `strict` mode, except skips missing required
```
passive voice fix

Reply:
@plumbis, validation is totally skipped at the moment, not just the "missing required schemas", the new wording makes me the latter.